oci/layout/oci_dest.go

```diff
@@ -178,7 +178,7 @@ func (d *ociImageDestination) PutManifest(m []byte) error {
 	}
 
 	annotations := make(map[string]string)
-	annotations["org.opencontainers.image.ref.name"] = d.ref.tag
+	annotations["org.opencontainers.image.ref.name"] = d.ref.image
```

Comment:
>  spec forbids it also

Where? I can’t see it; in fact the index specification says that all of `annotations` is optional.